CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 - Upgrade to `protoc` 3.19.2 support.
+- Add `buf beta registry plugin deprecate`, `buf beta registry plugin undeprecate`, `buf beta registry template deprecate` and `buf beta registry template undeprecate`.
```

Comment:
Fine as-is so don't feel obligated to change it, but we can make this message a little shorter if we wanted.
```suggestion
- Add `buf beta registry {plugin,template} {deprecate,undeprecate}`
```